docs/network-flow-visibility.md

```diff
@@ -983,165 +944,4 @@ by adding the file in the following section:
 ./hack/generate-manifest-flow-visibility.sh > build/yamls/flow-visibility.yml
 ```
 
-### ELK Flow Collector (deprecated)
```

Comment:
Maybe instead of removing this whole section, keep a disclaimer for a couple of releases. I think the whole section should be replaced with:

```
### ELK Flow Collector (removed)

**Starting with Antrea v1.7, support for the ELK Flow Collector has been removed.**
Please consider using the [Grafana Flow Collector](...) instead, which is actively maintained.
```

Reply:
added a disclaimer as suggested, thanks